.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,3 +23,5 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
 - For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
 - Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays.
 - Never force push. If you do, your PR will be closed.
+- Oppiabot can assign anyone for review/help if you leave a comment like the following: "{{Question/comment}} @{{reviewer_username}} PTAL"
```

Comment:
@DubeySandeep Just to check, will this actually trigger Oppiabot (by virtue of having an @-something in it) because it'll be in the first comment of every PR?

Reply:
I confirm that PR won't trigger bot ([ref](https://github.com/oppia/oppia/pull/11291#issuecomment-736765065)).

Should I add a test in the oppia-bot project to avoid such a case happen in the future?

Reply:
Created an issue to add test here: https://github.com/oppia/oppiabot/issues/174